src/js/Index.js

```diff
@@ -21,6 +21,7 @@ import { TimePicker } from './Framework/DateTimePicker/TimePicker'
 import Clipboard from './Framework/Clipboard'
 import Theme from './Framework/Theme'
 import SelectSearch from './Framework/SelectSearch'
+import passwordStrengthChecker from './Framework/PasswordStrengthChecker.js'
```

Comment:
Waarom is deze met kleine `p` en niet zoals de rest met een hoofdletter?

Reply:
Geen erg in gehad, ik pas het aan